os/kernel/init/os_bringup.c

```diff
@@ -264,6 +267,13 @@ static inline void os_do_appstart(void)
 	se_initialize();
 #endif
 
+#ifdef CONFIG_SECURITY_LEVEL
+	int ret = set_security_level();
+	if (ret != SECURITY_LEVEL_OK) {
+		dbg("[set_security_level] Failed to call set_security_level. ret : %d\n", ret);
```

Comment:
There are two `set_security_level` messages. Let's use once to save unnecessary flash usages and to optimize performance.